nixos/modules/services/networking/dnscrypt-proxy2.nix

```diff
@@ -0,0 +1,57 @@
+{ config, lib, pkgs, ... }: with lib;
+
+let
+  cfg = config.services.dnscrypt-proxy2;
```

Comment:
I think we should deprecate dnscrypt-proxy v1 for the next release. It is no longer maintained afaik.

Reply:
Agreed. v1 should be removed for 19.03